static/scss/voucher.scss

```diff
@@ -12,6 +12,15 @@
     color: $deep-sky-blue;
   }
 
+  .btn {
+    color: white;
+    border-radius: 0;
+    min-width: 180px;
+    font-weight: 600;
+    letter-spacing: 1.16px;
+    line-height: 26px;
+  }
+
```

Comment:
It seems like there should be an existing style for this button that we can reuse. I think it's the same as the "Apply" button on the checkout page and the "Home" button on the 404 page.

Reply:
mostly CSS is written context base, like here these styles are under `.voucher-container`. I can't find any exact match of this button.